api/database/migrations/data/2024_04_01_222259_create_draft_lyrics_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::connection('data')->create('draft_lyrics', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->uuid('track_id')->unique();
+            $table->json('document')->nullable();
```

Comment:
@szainmehdi  
Previously we had two separate columns, `content` and `format`
`content` was a JSON field to handle version 2 of lyrics, however format 1 was getting broken as we are not storing as JSON, but plain text, so merged the two fields into a field named `document`, this is similar to how its done in `Track` model however the column name in `Track` is `lyrics`

Are you happy with this change ? or you would like me to implement differently ?

Reply:
This works, good call!